src/openrct2/world/Location.hpp

```diff
@@ -503,6 +503,27 @@ constexpr Direction ALL_DIRECTIONS[] = { 0, 1, 2, 3 };
     return (dir - 1) & 0x03;
 }
 
+/**
+ * Given two positions, return the cardinal direction which is closest to the direction from 'from' to 'to'.
+ */
+[[maybe_unused]] static constexpr Direction direction_from_to(const CoordsXY& from, const CoordsXY& to)
```

Comment:
For all new functions make sure you use TitleCase

Reply:
You don't think it's better to be consistent with the other `direction_*` functions?

Reply:
No if anything it will serve as a reminder to fix them